### PR TITLE
CRV x cvxCRV Peg Test

### PR DESCRIPTION
**Pull request by icepaq**

### CODE REVIEW

File: contracts/Convex/AntecvxCRVPegTest.sol [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../AnteTest.sol";
+import "hardhat/console.sol";
```
Comment (icepaq) [inline contracts/Convex/AntecvxCRVPegTest.sol:6]: Remove

File: contracts/Convex/AntecvxCRVPegTest.sol [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        uint256 cvxCRVToETH = oneInchOracle.getRate(CVX_CURVE_ADDRESS, WETH_ADDRESS, false);
+
+        // At least 20 blocks should have passed since preCheck 
+        // No more than 40 blocks can pass between preCheck and checkTestPasses
+
+        if(preCheckBlock == 0 || preCheckSlip == 0) {
+            return true;
+        }
+
+        if (block.number - preCheckBlock > 40 || block.number - preCheckBlock < 10) {
```
Comment (icepaq) [inline contracts/Convex/AntecvxCRVPegTest.sol:52]: Remove upper bound limit

File: contracts/Convex/AntecvxCRVPegTest.sol [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            return true;
+        }
+
+        uint256 slip = (crvToETH * 100) / cvxCRVToETH;
+
+        // If each slippage test was within 10% of the preCheckSlip, then the test passes
+        if ((slip > 96 && slip < 104) && (preCheckSlip > 95 && preCheckSlip < 105)) {
+            return false;
```
Comment (icepaq) [inline contracts/Convex/AntecvxCRVPegTest.sol:60]: This should be true

File: contracts/Convex/AntecvxCRVPegTest.sol [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import "../AnteTest.sol";
+
+interface IOneInchOracle {
+    function getRate(address srcToken, address dstToken, bool useWrapper) external view returns(uint256);
+}
+
+/// @title Convex Curve Peg Test
+/// @notice The 1inch Oracle is unable to accurately price the two currencies to each other
+/// to get around this, we use the ETH exchange rate and compare the two.
+contract AntecvxCRVPegTest is AnteTest("Curve stETH Keeps 99% of it's ETH.") {
```
Comment (delalunia) [inline contracts/Convex/AntecvxCRVPegTest.sol:14]: Let's correct this title!

**Review by delalunia — APPROVED** [review]

**Pull request merged**